net-vpn/tor/tor-0.3.2.9.recipe

```diff
@@ -62,14 +69,23 @@ GLOBAL_WRITABLE_FILES="
 	settings/tor/torrc.sample auto-merge
 	"
 
+defineDebugInfoPackage tor$secondaryArchSuffix \
+	"$commandBinDir"/tor \
+	"$commandBinDir"/tor-gencert \
+	"$commandBinDir"/tor-resolve
+
 BUILD()
 {
 	export CFLAGS="-D_BSD_SOURCE"
 	export CXXFLAGS="-D_BSD_SOURCE"
 	export LDFLAGS="-lbsd"
+
 	autoreconf -vfi
-	runConfigure ./configure --disable-gcc-hardening
-	make
+
+	runConfigure --omit-dirs "binDir sbinDir" ./configure \
+		--bindir="$commandBinDir" --sbindir="$commandBinDir" \
+		--disable-gcc-hardening
```

Comment:
Since we now have proper support for SSP in gcc, should we stop disabling gcc hardening?

```diff
-		--disable-gcc-hardening
```

Reply:
yes.

Reply:
The build system doesn't link -lssp automatically (idk, why, maybe a bug in Haiku's gcc?). I can try to define it as LDFLAG hovewer.

Reply:
> The build system doesn't link -lssp automatically (idk, why, maybe a bug in Haiku's gcc?). I can try to define it as `LDFLAG` hovewer.

It does sounds like a bug to me, as other OSes doesn't require explicit declaration of `-lssp` in `LDFLAGS`

Reply:
It is possible that i made something wrong as I enabled libssp for gcc 5.4, maybe something missing.